### PR TITLE
feat: Configure the new container Row Category - EXO-61140 - Meeds-io/MIPs#37

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy [hunk 1]
```diff
@@ -26,6 +26,12 @@ import org.exoplatform.webui.core.model.SelectItemOption ;
     table.addSelectItemOption(new SelectItemOption("simpleTable",
         "<container template=\"system:/groovy/portal/webui/container/UISimpleTableContainer.gtmpl\"></container>",
         "SimpleTableContainerLayout"));
-  templates.add(table)
+  templates.add(table);
+  
+  SelectItemCategory Row = new SelectItemCategory("SimpleRowContainer");
```
Comment (azayati) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]:
```suggestion
  SelectItemCategory Row = new SelectItemCategory("row");
```

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy [hunk 2]
```diff
@@ -26,6 +26,12 @@ import org.exoplatform.webui.core.model.SelectItemOption ;
...
+    Row.addSelectItemOption(new SelectItemOption("SimpleRowContainerItem",
```
Comment (azayati) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]:
```suggestion
    Row.addSelectItemOption(new SelectItemOption("simpleRow",
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy (hunk 2 above)
Comment (sofyenne) [reply commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]: done !

**Review by azayati — APPROVED** [review]
LGTM

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy [hunk 3]
```diff
@@ -26,6 +26,12 @@ import org.exoplatform.webui.core.model.SelectItemOption ;
     table.addSelectItemOption(new SelectItemOption("simpleTable",
         "<container template=\"system:/groovy/portal/webui/container/UISimpleTableContainer.gtmpl\"></container>",
         "SimpleTableContainerLayout"));
-  templates.add(table)
+  templates.add(table);
+  
+  SelectItemCategory Row = new SelectItemCategory("row");
```
Comment (azayati) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]:
```suggestion
  SelectItemCategory row = new SelectItemCategory("row");
```

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy [hunk 4]
```diff
@@ -26,6 +26,12 @@ import org.exoplatform.webui.core.model.SelectItemOption ;
...
         "<container template=\"system:/groovy/portal/webui/container/UISimpleTableContainer.gtmpl\"></container>",
         "SimpleTableContainerLayout"));
-  templates.add(table)
+  templates.add(table);
+  
+  SelectItemCategory Row = new SelectItemCategory("row");
+    Row.addSelectItemOption(new SelectItemOption("simpleRow",
+        "<container template=\"system:/groovy/portal/webui/container/UISimpleRowContainer.gtmpl\"></container>",
+        "SimpleRowContainerLayout"));
+  templates.add(Row);
```
Comment (azayati) [inline commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]:
```suggestion
  templates.add(row);

```

**Review by azayati — CHANGES_REQUESTED** [review]

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy (hunk 4 above)
Comment (sofyenne) [reply commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]: done !

File: commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy (hunk 3 above)
Comment (sofyenne) [reply commons-extension-webapp/src/main/webapp/WEB-INF/conf/uiconf/portal/webui/container/ContainerConfigOption.groovy]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**